openapi.yaml

```diff
@@ -848,7 +857,7 @@ components:
         DFIAccountNumber:
           type: string
           description: |
-            The receiver's bank account number you are crediting/debiting. It important to note that this is an alphanumeric field, so its space padded, no zero padded
+            The receiver's bank account number you are crediting/debiting. It important to note that this is an alphanumeric field, so it's space padded, not zero padded
```

Comment:
nit: should there be `-` before `padded`?

Reply:
I've seen both ways.

Reply:
"Generally, you need the hyphen only if the two words are functioning together as an adjective before the noun they’re describing. If the noun comes first, leave the hyphen out." - Grammarly

Since space padded comes after the noun in this case, I don't think it needs a hyphen